openlibrary/catalog/marc/parse.py

```diff
@@ -1,4 +1,5 @@
 import re
+from typing import Optional
```

Comment:
[Open Library runs on Python 3.11](https://openlibrary.org/status) so the `str | None` syntax is also acceptable.

Reply:
Thanks for the suggestion. Sorry I didn't get the fix in before the merge, but I'll remember the hint for next time.